cli/command/stack/common.go

```diff
@@ -0,0 +1,32 @@
+package stack
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+)
+
+// validateStackName checks if the provided string is a valid stack name (namespace).
+//
+// It currently only does a rudimentary check if the string is empty, or consists
+// of only whitespace and quoting characters.
+func validateStackName(namespace string) error {
+	v := strings.TrimFunc(namespace, quotesOrWhitespace)
+	if len(v) == 0 {
```

Comment:
nit: isn't `if v == ""` clearer?

Reply:
yes it is (same, copy/pasta, but I'll update :wink: )